File: app/general/Filters.java
package general;

import javax.inject.Inject;

import play.filters.headers.SecurityHeadersFilter;
import play.http.DefaultHttpFilters;

/**
 * Increases security
 * (https://www.playframework.com/documentation/2.5.x/SecurityHeaders)
 */
public class Filters extends DefaultHttpFilters {

	@Inject
	public Filters(SecurityHeadersFilter securityHeadersFilter) {
		super(securityHeadersFilter);
	}
}

File: app/general/Filters.scala
package general

import javax.inject.Inject

import play.api.http.DefaultHttpFilters
import play.filters.headers.SecurityHeadersFilter

/**
  * Increases security
  * (https://www.playframework.com/documentation/2.5.x/SecurityHeaders)
  */
class Filters @Inject() (securityHeadersFilter: SecurityHeadersFilter)
  extends DefaultHttpFilters(securityHeadersFilter)
